Components/BlockMediaText/howto.md

```diff
@@ -26,3 +26,16 @@ add_filter('Flynt/addComponentData?name=BlockMediaText', function ($data) {
     }
 });
 ```
+
+**Beware:** If you use this component as a static component (or any component with tabs), you need to add a closing tab*, otherwise the next components will show up inside the last tab.
+
+*A tab with an empty label, and `"endpoint": 1`
+
+```json
+{
+  "name": "endTab",
+  "label": "",
+  "type": "tab",
+  "endpoint": 1
+}
+```
```

Comment:
Actually, could you check if it's enough to set the last used tab to `"endpoint": 1`? Maybe this way we don't need this extra tab and don't have any issues no matter how the component is used...